Jellyfin.Plugin.ThePornDB/Configuration/PluginConfiguration.cs

```diff
@@ -56,11 +68,11 @@ public PluginConfiguration()
             this.UseUnmatchedTag = false;
             this.UnmatchedTag = "Missing From ThePornDB";
 
-            this.DisableMediaAutoIdentify = false;
-            this.DisableActorsAutoIdentify = false;
+            this.AutoIdentifyOptionMedia = AutoIdentifyOptionMedia.Enable;
+            this.AutoIdentifyOptionActors = AutoIdentifyOptionActors.Enable;
```

Comment:
I don't want to people update their config (backward compatibility), what is done is done (I don't like old name too)